.vscode/launch.json

```diff
@@ -110,6 +110,24 @@
       ]
     },
 
+    // Neovim launch configs
+    {
+      "name": "Neovim: Run",
+      "request": "attach",
+      "continueOnAttach": true,
```

Comment:
Have you tried `"outputCapture": "std",`? Seems like a longshot, but maybe that would capture debug output? Seems possibly relevant anyway

Reply:
I have just tried it and I don't see any difference/new stuff being logged/printed